app/helpers/provider_helper.rb

```diff
@@ -64,4 +64,8 @@ def google_form_url_for(settings, email, provider)
   def is_current_cycle(cycle_year)
     Settings.current_recruitment_cycle_year == cycle_year.to_i
   end
+
+  def provider_code_or_code(params)
+    params[:code] || params[:provider_code]
```

Comment:
should it be just `params[:code]`

Given the route

Reply:
https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/ad8233a36d6fae55817fec06cae53e35f5005ccc